code/controllers/subsystem/persistence.dm

```diff
@@ -272,6 +284,25 @@ SUBSYSTEM_DEF(persistence)
 	fdel(json_file)
 	WRITE_FILE(json_file, json_encode(file_data))
 
+/datum/controller/subsystem/persistence/proc/CollectDynamicRules()
+	var/list/amount_of_rules = length(CONFIG_GET(number_list/repeated_mode_adjust))
+	var/datum/game_mode/dynamic/dynamic = SSticker.mode
+	if(istype(dynamic))
+		var/list/this_round = list()
+		for(var/datum/dynamic_ruleset/roundstart/roundstart_rule in dynamic.executed_rules)
+			if(!CHECK_BITFIELD(roundstart_rule.flags, HIGH_IMPACT_RULESET|ONLY_RULESET|NO_OTHER_ROUNDSTARTS_RULESET))
```

Comment:
This should be its own flag, `PERSISTENT_RULESET`.

Reply:
Done.